### PR TITLE
New package: mabel-0.1.3

### PR DESCRIPTION
**Pull request by FIGBERT**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-musl
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64
  - aarch64-musl
  - armv7l

### CODE REVIEW

File: srcpkgs/mabel/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+pkgname=mabel
+version=0.1.3
+revision=1
+build_style=go
+go_import_path="github.com/smmr-software/${pkgname}"
+short_desc="Fancy BitTorrent client for the terminal"
+maintainer="SMMR Software <hello@smmr.software>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/smmr-software/mabel"
+distfiles="https://github.com/smmr-software/${pkgname}/archive/refs/tags/v${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/mabel/template:11]: expand $pkgname here

**Review by classabbyamp — COMMENTED** [review]
does this need a go ld flag for setting the version?

File: srcpkgs/mabel/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'mabel'
+pkgname=mabel
+version=0.1.3
+revision=1
+build_style=go
+go_import_path="github.com/smmr-software/${pkgname}"
+go_ldflags="-s -w -X main.version=${version} -X main.builtBy=xbps -X main.date=$(date --iso-8601)"
```
Comment (classabbyamp) [inline srcpkgs/mabel/template:7]: use this for setting the date instead:

https://github.com/void-linux/void-packages/blob/ddd3c3cd8ab4e045009f74978ff63dce76660974/srcpkgs/chezmoi/template#L16-L22

File: srcpkgs/mabel/template [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'mabel'
+pkgname=mabel
+version=0.1.3
+revision=1
+build_style=go
+go_import_path="github.com/smmr-software/${pkgname}"
```
Comment (Duncaen) [inline srcpkgs/mabel/template:6]:
```suggestion
go_import_path="github.com/smmr-software/mabel"
```

The package name is not variable, and the variable is longer than the actual name.

**Comment by FIGBERT** [comment]
Any updates?

**Comment by FIGBERT** [comment]
I'm hopeful that this will be merged eventually, so here is me bumping it.

**Pull request closed without merging**